app/src/main/java/org/eyeseetea/malariacare/database/AppDatabase.java

```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright (c) 2015.
+ *
+ * This file is part of QIS Survelliance App.
+ *
+ *  QIS Survelliance App is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  QIS Survelliance App is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
@idelcano I think this Foobar should be removed with the name of the app as well. I have just realised it is wrong for as well in malariapp